src/Wallabag/ImportBundle/Import/WallabagV1Import.php

```diff
@@ -141,6 +142,14 @@ protected function parseEntries($entries)
             $entry->setArchived($importedEntry['is_read']);
             $entry->setStarred($importedEntry['is_fav']);
             $entry->setContent($importedEntry['content']);
+            if ($importedEntry['tags'] != '') {
+                $tags = explode(',', $importedEntry['tags']);
+                foreach ($tags as $tagv1) {
+                    $tag = new Tag();
```

Comment:
You should check if tag exist first. See https://github.com/wallabag/wallabag/blob/v2/src/Wallabag/ImportBundle/Import/PocketImport.php#L180

We need this method to be called from (almost) everywhere since we already use it in ContentProxy.